README.md

```diff
@@ -53,8 +53,8 @@ Building Skulpt is straightforward:
 
 1. Clone the repository from GitHub, ideally using your own fork if you're planning on making any contributions
```

Comment:
We should add here an hint, that Python 2.7ish is required. The npm commands may delude new users, that no Python is required for building.